containers/allan_dip_bot/run_bot.py

```diff
@@ -274,6 +282,13 @@ def main() -> None:
     parser.add_argument(
         "--outdir", type=Path, help="output directory for game json to be stored"
     )
+    parser.add_argument(
+        "--aggressiveness",
+        type=str,
+        choices=[str(a.value) for a in Aggressiveness],
+        default="moderate",
+        help='aggressiveness of the bot ("aggressive", "moderate", "friendly"). Default is "moderate".',
+    )
     args = parser.parse_args()
```

Comment:
That's a good way to implement `choices`! I have some other suggestions:

```suggestion
    parser.add_argument(
        "--aggressiveness",
        type=str,
        choices=[str(a.value) for a in Aggressiveness],
        default=Aggressiveness.moderate.value,
        help='aggressiveness of the bot (default: %(default)s)',
    )
    args = parser.parse_args()
```

You don't need to list choices because `argparse` does it automatically. There is also templating so the default doesn't need to be repeated as well.

Reply:
done